internal/rbd/controllerserver.go

```diff
@@ -833,6 +878,48 @@ func (cs *ControllerServer) DeleteVolume(
 	}
 	defer cs.OperationLocks.ReleaseDeleteLock(volumeID)
 
+	if isMigrationVolHandle(volumeID) {
+		log.DebugLog(ctx, "migration volume handle : %s", volumeID)
+		pMH, pErr := parseMigrationVolHandle(volumeID)
+		if pErr != nil {
+			log.DebugLog(ctx, "failed to parse migration volume handle: %s , err: %v", volumeID, pErr)
+
+			return nil, pErr
+		}
+		rv := &rbdVolume{}
+		if pMH != nil {
+			rv.RbdImageName = pMH.ImageName
+			rv.Monitors = pMH.Mons
+			rv.Pool = pMH.Pool
+		}
+		log.DebugLog(ctx, "filled rbd volume: %+v", rv)
+		monsMap := make(map[string]string)
+		monsMap["monitors"] = rv.Monitors
+		clusterID, cErr := getClusterIDFromMigrationVolume(monsMap)
```

Comment:
why do you need clusterID here?

Reply:
`deleteImage->..->addRbdManagerTask()` use the `clusterID`.

Reply:
still, I don't see that code where we are using the clusterID. can you please point me to the code where the clusterID is getting used?